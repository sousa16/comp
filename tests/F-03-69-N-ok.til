(program
  (int x 1)
  (loop (<= x 3) (block
    (int y 3)
    (println "I	ER x=" x)
    (loop (>= y 0) (block
      (println y)
      (if (&& (== y 2) (== x 2)) (stop 2))
      (set y (- y 1))
    ))
    (set x (+ x 1))
  ))
  (println x)
  (return 0)
)

