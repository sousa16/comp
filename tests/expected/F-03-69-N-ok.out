I	ER x=1
3
2
1
0
I	ER x=2
3
2
2
